homeassistant/components/monarchmoney/strings.json

```diff
@@ -0,0 +1,49 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Enter your Monarch Money email and password, if required you will also be prompted for your MFA code.",
+        "data": {
+          "mfa_secret": "Add your MFA Secret. See docs for help.",
+          "mfa_code": "Enter your MFA code",
+          "email": "[%key:common::config_flow::data::email%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      }
+    },
+    "error": {
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "mfa_required": "Multi-factor authentication required.",
+      "bad_mfa": "Your code was invalid, please try again or use a recovery token."
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "balance": { "name": "Balance" },
+      "value": { "name": "Value" },
+
+      "age": {
+        "name": "Data age"
+      },
+
+      "sum_income": {
+        "name": "Income YTD"
+      },
+      "sum_expense": {
+        "name": "Expense YTD"
+      },
+      "savings": {
+        "name": "Savings YTD"
```

Comment:
YTD may not be common in other languages cultures, so maybe write it out?

Reply:
Isn't that what translations are for? Also MonarchMoney at this point only supports dollars / USD ... but ok

Reply:
I mean, it means Year to date, but that doesn't have a proper dutch equivalent for example